charts/thoras/templates/collector/service.yaml

```diff
@@ -11,3 +11,16 @@ spec:
     targetPort: {{ .Values.metricsCollector.search.containerPort }}
   selector:
     app: metrics-collector
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.metricsCollector.timescale.name }}
```

Comment:
[pebble] Any reason we're giving the user the option to name this? [I don't believe we've historically done that for other components](https://github.com/thoras-ai/helm-charts/blob/main/charts/thoras/templates/dashboard/service.yaml#L5) -- feels a bit like YAGNI and unnecessary config complexity to me personally. I'm not super strong on it tho -- it's working software